ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -78,8 +85,13 @@ describe('ActivitiPeople', () => {
     }));
 
     it('should show people component title', () => {
-        expect(element.querySelector('#people-title')).toBeDefined();
-        expect(element.querySelector('#people-title')).not.toBeNull();
+        activitiPeopleComponent.people = [...userArray];
```

Comment:
i don't think the test is testing correctly the behaviour without the async clause here.